script.js

```diff
@@ -0,0 +1,51 @@
+const nav = document.querySelector('.page-flow');
+const ham = document.querySelector('.hamburger');
+const body = document.querySelector('body');
+const li = document.querySelectorAll('.page-flow > li');
+const x = window.matchMedia('(max-width: 768px)');
+const sec = document.querySelectorAll('*');
+let id;
+
+function myFunction(x) {
+  // If media query matches
+  if (x.matches) {
+    x.addListener(myFunction);
```

Comment:
- [ ]  Hey, this chained statement is preventing your mobile menu from showing up, I fixed it from my end and it worked as expected. kindly double-check the statement closely to fix this issue.
- [ ] I notice your background color is not the same as in the Figma design. Please, kindly ensure you've used the correct background to match the design after fixing the issue stated above


*Figma*
![Screenshot from 2022-12-19 20-48-20](https://user-images.githubusercontent.com/59397159/208508737-b4a8c3b3-ce3d-4bf0-a362-509379f13095.png)


*yours*
![Screenshot from 2022-12-19 20-47-39](https://user-images.githubusercontent.com/59397159/208508741-11d27e94-8303-41a8-8d7c-49c3ad3e1eaf.png)

Reply:
@zieeco I have fixed it, but using another method. Can you please tell me how you did it with this existing one?
I tried a couple of things, but couldn't reach a solution as it was always failing some test case.